public/index.html

```diff
@@ -8,20 +8,31 @@
 <body>
   <div id="app"></div>
 
-
   <!-- React -->
   <!-- Note: when deploying, replace "development.js" with "production.min.js". -->
   <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
   <script>
   //Pull Search Params
   var params = new URLSearchParams(window.location.search);
-  var houseID = params.get('id')
+  var houseID = params.get('id');
   // Request for local map
-  var scriptUrl = `http://127.0.0.1:8002/proxy?id=${houseID}`
+  var mapScriptUrl = `http://127.0.0.1:8002/proxy?id=${houseID}`;
   var localMapScript = document.createElement('script');
-  localMapScript.setAttribute('src', scriptUrl);
+  localMapScript.setAttribute('src', mapScriptUrl);
   document.body.append(localMapScript)
+
+  //Gallery Module
+  var galleryScriptUrl = `http://127.0.0.1:8001/bundle.js?id=${houseID}`;
+  var galleryScript = document.createElement('script');
+  galleryScript.setAttribute('src', galleryScriptUrl);
+  document.body.append(galleryScript);
+
+  //Listings module
+  var nearbyListingsUrl = `http://127.0.0.1:8003/bundle.js?id=${houseID}`;
+  var nearbyListingsScript = document.createElement('script');
+  nearbyListingsScript.setAttribute('src', nearbyListingsUrl);
+  document.body.append(nearbyListingsScript);
```

Comment:
Is this how you were able to get the id in the search bar to work with all the modules? Because importing just bundle.js as the source in the script tag worked for my module on my proxy. Just curious if this was necessary.

Reply:
Maybe it's not? I'll check. My logic might be flawed here.

Reply:
They weren't necessary. Made the fixes, thank you for the recommendations.